source/ed/docker_tests/ed_integration_tests.cpp

```diff
@@ -378,7 +378,7 @@ BOOST_FIXTURE_TEST_CASE(ntfs_v5_test, ArgsFixture) {
             boost::gregorian::date_period("20150826"_d, "20150926"_d));
     BOOST_CHECK_EQUAL(data.pt_data->datasets[0]->realtime_level == nt::RTLevel::Base, true);
     BOOST_CHECK_EQUAL(data.pt_data->datasets[0]->system, "obiti");
-    BOOST_CHECK_EQUAL(data.pt_data->vehicle_journeys[0]->dataset->uri, "d1");
+    BOOST_CHECK(data.pt_data->vehicle_journeys[0]->dataset == nullptr);
```

Comment:
Can you add a comment to that, so we know that it's not an enforced behavior, it's never supposed to happen.
Something like:
`// as ntfs_v5 fixture does not contain datasets.txt which is now required, we shouldn't have a dataset linked to vj`
That way, if someone implements a fallback he will have no remorse changing the test.